PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+*More detailed description of your change,
```

Comment:
Would we want to add a line item for suggestions on the title?

Reply:
This would mean adding something like *KAFKA-1234: Short summary of the change.* at the top of this file.

Reply:
Does the first line of the template appear in the PR title?

Reply:
oh. I see from your comment below that this is what the PR box would. I know the first line of the commit message appears as the PR title, but not for the actual text in the description box of the PR page. 

Is there a way to have a template for the title?

Reply:
I need to check, it's a good question and would be good to do it, if possible.

Reply:
I couldn't find a way to set the title from the template.